lib/slack/web/web.ex

```diff
@@ -57,6 +57,9 @@ Enum.each(Slack.Web.get_documentation, fn({module_name, functions}) ->
       end
     end)
 
+    defp get_token(%{token: token}) when is_binary(token), do: token
```

Comment:
`is_binary` was originally requested by @davydog187 and is consistent with the token type. Feel free to modify the code, tho.